sdk/precompiles/src/lib.rs

```diff
@@ -0,0 +1,114 @@
+use {
```

Comment:
nit: probably not needed, but
```suggestion
#![cfg_attr(docsrs, feature(doc_auto_cfg))]
use {
```